salesforce-analytics-cloud/3.2/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,397 @@
+* Keystore File +
+Path to the keystore used to sign data during authentication. NOTE: Only the Java keystore format is allowed.
+* Store Password +
+Keystore password.
+* Principal +
+Salesforce username.
+
+image::salesforce-analytics-saml.png[SAML Bearer Configuration]
+
+[[generating-keystore]]
+==== Generating a Keystore File
+
+The Keystore is the path to the keystore used to sign data during authentication. This example shows how to generate a JKS file.
+
+To generate a keystore file:
+
+. Go to your Mule workspace, and open the command prompt (for Windows) or Terminal (for Mac).
+. Type `keytool -genkeypair -alias salesforce-cert -keyalg RSA -keystore salesforce-cert.jks` and press enter.
+. Enter the following details:
++
+** Password for the keystore.
+** Your first name and last name.
+** Your organization unit.
+** Name of your city, state, and the two letter code of your county.
++
+The system generates a java keystore file containing a private/public key pair in your workspace.
++
+. Provide the file path for the keystore in your connector configuration.
++
+Type `keytool -exportcert -alias salesforce-cert -file salesforce-cert.crt -keystore salesforce-cert.jks` and press enter.
++
+The system now exports the public key from the keystore into the workspace. This is the public key that you need to enter in your Salesforce instance.
++
+. Verify that you have both the keystore (`salesforce-cert.jks`)
+and the public key (`salesforce-cert.crt`) files in your workspace.
+
+== Add the Connector in Studio
+
+. In Studio, create a Mule project.
+. In the Mule Palette view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type "analytics" in the search field.
+. Click this connector's name in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+=== Configure the Connector in Studio
+
+. Drag the desired Salesforce Analytics operation to the Studio canvas.
+. To create a global element for the connector, on the *General* tab, configure the authentication as described in the authentication sections.  
++
+The following authentication connection types are available: +
++
+* <<basic-authentication,Username Password>>
+* <<oauth2,OAuth 2.0>>
+* <<oauth2-jwt,OAuth 2.0 JWT>>
+* <<oauth2-saml,OAuth 2.0 SAML Bearer>>
+. Configure these fields to upload external data into new dataset and start the processing operation:
+** Type +
+Type of the records to insert. Select a JSON file representing the schema of the dataset to be created.
+** Records +
+DataSense expression (the records to be inserted).
+** Operation +
+Specify the operation to use when you’re loading data into a dataset.
+** Description
+** Label
+** Data Set Name
++
+image::salesforce-analytics-studio-upload-and-start.png[Upload external data]
+
+[[basic-authentication]]
+=== Username Password
+
+Enter the following information on the *General* tab of the *Global Element Properties* screen to configure Basic authentication:
```

Comment:
```suggestion
On the *General* tab of the *Global Element Properties* screen, enter the following information to configure Basic authentication:
```

Reply:
Same with the rest sections, but this is just me that I like to locate first the section and then the action...you can also leave it as it is.

Reply:
I actually agree with you on that.